src/options.h

```diff
@@ -2,6 +2,8 @@
 
 class Options {
 	public:
-		Options() {}
-		short getPort(const char* file_name);
+		Options(const char* file_name);
+		short getPort();
+	private:
+		NginxConfig config;
```

Comment:
minor convention: add a trailing `_` to member variables, maybe change this in the future if you get a chance haha